src/tests/ftest/deployment/agent_failure.yaml

```
@@ -46,8 +46,15 @@ ior:
     # This test doesn't work if the IOR runs too fast. In that case, increase
     # block_size and/or decrease transfer_size. To decrease transfer_size, divide it
     # by 2, 4, 8, etc.
-    transfer_size: 256K
+    transfer_size: 256K  # CI
+    # transfer_size: 1G  # Aurora
     block_size: 100G
     sw_deadline: 60
-    dfs_oclass: SX
-    dfs_dir_oclass: SX
+ior_wo_rf:
+  <<: *ior_base
+  dfs_oclass: SX
+  dfs_dir_oclass: SX
+ior_with_ec:
+  <<: *ior_base
+  dfs_oclass: EC_16P2G1
+  dfs_dir_oclass: EC_16P2G1
```

Better to use `X` for single shared file. Is there a reason for using `EC_16P2G1`? Also, `dfs_dir_oclass` is unused for IOR.
```suggestion
ior_wo_rf:
  <<: *ior_base
  dfs_oclass: SX
ior_with_ec:
  <<: *ior_base
  dfs_oclass: EC_16P2GX
```

---

I updated the code to use `EC_2P2GX` in CI and `EC_16P2GX` in Aurora (I think we need more than 3 nodes to use `EC_16P2GX`). Also noticed that if I remove `dfs_dir_oclass`, it's automatically added by test harness and uses SX.
```
ior -a DFS -b 100G -k -v -w -W -D 60 -o /test_file_1 -t 256K --dfs.chunk_size 1048576 --dfs.cont TestContainer_1 --dfs.dir_oclass SX --dfs.oclass EC_2P2GX --dfs.pool TestPool_1
```
I'm not sure if this okay.

---

Yeah, the framework shouldn't be doing that. But it doesn't matter. `ior` literally doesn't use `dir_oclass` at all

---

Also, EC16P2 needs at least 16+2 nodes, + number of failures

---

That's because it's shared with `mdtest`, which does use it :)

---

ah I see. Thanks.